server/src/server.js

```diff
@@ -576,7 +579,7 @@ app.post("/api/login", function(req, res) {
   });
 });
 
-app.put("/data/:id/:domain", function(req, res) {
+app.put("/data/:id/:domain", upload.single('blob'), function(req, res) {
```

Comment:
After thinking about it, just having the `upload.single` middleware probably doesn't break the tests, but something else.